package.json

```diff
@@ -3,21 +3,19 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "clean": "pnpm nx run-many --target=clean --all",
```

Comment:
Can we still keep this command? with `pnpm -r` instead.
It's helpful for when people did something wrong, I can see it being useful already for people do the transition when this branch gets merged.

Reply:
I have replaced this with the "simulate-fresh-git-clone" script which does the same but better and more complete. This command here only removes some of the artifacts and actually misses quite a lot of other ones like cache files and out dirs etc. My script uses every gitignore in the repo to do this task, making sure it removes everythihng. We could rename that script to be just "clean" and it would have the same result as here.

Also we don't want this to only run inside what nx considers to be packages and projects, we want it to run for the whole monorepo regardless of nx or pnpm or anything else.